Gemfile.lock

```diff
@@ -20,6 +20,12 @@ GEM
       erubis (~> 2.7.0)
       rails-dom-testing (~> 1.0, >= 1.0.5)
       rails-html-sanitizer (~> 1.0, >= 1.0.1)
+    active-triples (0.8.1)
```

Comment:
Are there objects to a future pull request removing the Gemfile.lock and adding it to the .gitignore? I've always found it a hassle in collaborative development.